net.openra.OpenRA.json

```diff
@@ -0,0 +1,328 @@
+          "url": "https://github.com/OpenRA/SDL2-CS/releases/download/20161223/SDL2-CS.dll.config",
+          "dest-filename": "thirdparty/download/SDL2-CS.dll.config",
+          "sha256": "eff0594f7e86d40a94c1b28f8db87478f3de1cf0857f3d46e384bcc6744ce094"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/OpenRA/OpenAL-CS/releases/download/20151227/OpenAL-CS.dll",
+          "dest-filename": "thirdparty/download/OpenAL-CS.dll",
+          "sha256": "d33391fd954c670a3ba3db2d1553015f78652ea33ca3eaf949da6377ec947e53"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/OpenRA/OpenAL-CS/releases/download/20151227/OpenAL-CS.dll.config",
+          "dest-filename": "thirdparty/download/OpenAL-CS.dll.config",
+          "sha256": "58eaf46a749e466fce0a0ec43184073b0153aa8eb24c865219491bc1db989571"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/OpenRA/Eluant/releases/download/20160124/Eluant.dll",
+          "dest-filename": "thirdparty/download/Eluant.dll",
+          "sha256": "218c4ea5424b44d746b5343563123beebc652fd9731967dfd49fed071b58df31"
+        },
+        {
+          "type": "file",
+          "url": "https://nuget.org/api/v2/package/SmartIrc4net/0.4.5.1",
+          "dest-filename": "thirdparty/download/SmartIrc4net.zip",
+          "sha256": "4b58596540383f58a24a99cf73391b3e80e15d7498dec785ac7cbb34a40ea3b4"
+        },
+        {
+          "type": "file",
+          "url": "https://nuget.org/api/v2/package/rix0rrr.BeaconLib/1.0.1",
+          "dest-filename": "thirdparty/download/rix0rrr.BeaconLib.zip",
+          "sha256": "e8e126b98ae7676cb0434455aa8a41c17a927c1b40c4cf368f9527af6bc251f2"
+        },
+        {
+          "type": "archive",
+          "url": "https://geolite.maxmind.com/download/geoip/database/GeoLite2-Country_20171107.tar.gz",
+          "dest": "thirdparty/download/",
+          "sha256": "ad497e127835fd5af30aa512ab063e36b561aaa5f91281b1b92ad089b7c659ec"
+        },
+        {
+          "type": "shell",
+          "commands": ["gzip thirdparty/download/GeoLite2-Country.mmdb"]
+        },
+        {
+          "type": "patch",
+          "path": "OpenRA-lua-search-path.patch"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "make version",
+        "make dependencies",
+        "make",
+        "make prefix=/app install",
+        "make prefix=/app install-linux-shortcuts",
+        "make prefix=/app install-linux-appdata"
+      ]
+    },
+    {
+      "name": "export-other-mods",
+      "buildsystem": "simple",
+      "build-commands": [
+        "sed -i 's/<name>OpenRA - Red Alert/<name>OpenRA/' /app/share/appdata/openra-ra.appdata.xml",
+        "sed -i 's/<id>openra/<id>net.openra.OpenRA.openra/' /app/share/appdata/openra-{cnc,d2k}.appdata.xml",
+        "sed -i 's/Icon=openra/Icon=net.openra.OpenRA.openra/' /app/share/applications/openra-{cnc,d2k}.desktop",
```

Comment:
I think it's being fixed automatically (but I admit that I can be wrong about it).

Reply:
It's not fixed automatically.